backend/src/api/model/event.rs

```diff
@@ -135,6 +136,25 @@ impl Event {
     }
 }
 
+#[derive(juniper::GraphQLUnion)]
+#[graphql(Context = Context)]
+pub(crate) enum EventById {
+    Event(Event),
+    NotAllowed(NotAllowed),
+}
+
+impl EventById {
```

Comment:
> `Series.events` and all endpoints that return multiple events just omit the ones you can't see.

Ah right, that makes sense.

> And re `impl Into`... maybe but I don't see an advantage here right now. And it's not per se unidiomatic to not use `Into` for conversion functions.

Aight, I trust you. xP Out of curiosity, though: Is there any reason **not** to do it? Like, what's your rationale here?

Reply:
Mh, I'm surprised I can't find anything like that in the docs, but for me `Into` and `From` are for "obvious/trivial" and "no surprises" conversions. Everything that might need explanation or is only one of several ways you could do the conversion, I do prefer extra methods for. But hey, maybe that's also not really relevant. We are also not building a library here, so that's maybe different. 
Dunno, no really good reason, but I think both is fine.

Reply:
Mh, I'm surprised I can't find anything like that in the docs, but for me `Into` and `From` are for "obvious/trivial" and "no surprises" conversions. Everything that might need explanation or is only one of several ways you could do the conversion, I do prefer extra methods for. But hey, maybe that's also not really relevant. We are also not building a library here, so that's maybe different. 
Dunno, no really good reason, but I think both is fine.

Reply:
Mmmh okay, fair.